src/pages/Models/Details/_model-details.scss

```diff
@@ -4,34 +4,31 @@
 .model-details {
   display: grid;
   grid-template-columns: 230px 1fr;
+  margin-bottom: 2rem;
   padding-bottom: 14rem;
 
-  @media (min-width: $breakpoint-m-large) {
-    grid-template-columns: 350px 1fr;
-  }
-
-  &__header {
-    align-items: center;
-    display: grid;
-    grid-template-columns: 0.5fr 3fr 3rem;
-    padding-left: 1rem;
+  &:last-child {
+    margin-bottom: 1rem;
   }
 
-  &__main {
-    padding-top: 1rem;
+  td,
+  th {
+    &:first-child {
+      padding-left: 0.5rem;
+    }
   }
 
-  &__filters {
-    display: flex;
-    margin-left: auto;
+  tr {
+    display: grid;
   }
 
-  &__user {
-    text-align: center;
-  }
+  tbody tr {
+    @include vf-animation(background, brisk, ease-in-out);
 
-  tr {
-    display: grid;
+    &.is-selected,
+    &.is-selected:hover {
+      background-color: #cef3ff;
```

Comment:
Any good reason we don't use a Vanilla color here?

Reply:
This colour wasn't defined in the settings overrides and I didn't see it here: https://docs.vanillaframework.io/settings/color-settings so I left it. Which variable should I define/override for it?

Reply:
I merged this, we can address this fix in a follow-up.